config/config.go

```diff
@@ -624,7 +623,6 @@ var defaultConf = Config{
 	RunDDL:                       true,
 	SplitTable:                   true,
 	Lease:                        "45s",
-	TokenLimit:                   1000,
 	OOMUseTmpStorage:             true,
```

Comment:
There should be a line in config.go for deprecated variables. If you add "token-limit" to it, it will mean a cleaner error for users.